safe_transaction_service/history/migrations/0068_alter_multisigtransaction_origin.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 4.1.2 on 2022-11-14 18:39
+
+import json
+
+from django.db import migrations, models
+
+
+def convert_str_to_json(apps, schema_editor):
+    MultisigTransaction = apps.get_model("history", "MultisigTransaction")
+    transactions = (
+        MultisigTransaction.objects.exclude(origin__icontains="{")
+        .filter(origin__isnull=False)
+        .iterator()
+    )
+    for transaction in transactions:
+        transaction.origin = json.dumps(transaction.origin)
```

Comment:
- Shouldn't this be `json.loads`? You have a string and you want to load it as a Json. Is this tested? Unless `JSONField` is doing some conversion, you are just storing `str`.
- Users can store anything on this field, and `origin__icontains="{"` is not a good validation, so you should check for exceptions when loading the json
- Empty strings should be stored as `{}`
- Maybe we can try https://github.com/wemake-services/django-test-migrations

Reply:
> Shouldn't this be json.loads? You have a string and you want to load it as a Json. Is this tested? Unless JSONField is doing some conversion, you are just storing str.

It has mentioned here https://github.com/safe-global/safe-transaction-service/pull/1177#discussion_r1021943193

> Users can store anything on this field, and origin__icontains="{" is not a good validation, so you should check for exceptions when loading the json

It's an exclude the `alterJsonField` should convert any json string to json without issues. 

> Empty strings should be stored as {}

I agree I will add it in the migration. 

> Maybe we can try https://github.com/wemake-services/django-test-migrations

Sounds good, lets me try.